apps/meteor/client/startup/customOAuth.ts

```diff
@@ -10,15 +11,16 @@ Meteor.startup(() => {
 		})
 		.observe({
 			async added(record) {
-				const { isOauthCustomConfiguration } = await import('@rocket.chat/rest-typings');
-				if (!isOauthCustomConfiguration(record)) {
+				const service = record as unknown as (ILoginServiceConfiguration & OAuthConfiguration) | undefined;
```

Comment:
note: this typecast is temporary, the whole query is being replaced in another PR.